### PR TITLE
Changelog for 067

### PR DESCRIPTION
**Pull request by kelset**
Generated via the script, from 0.66.1 to 0.67-rc.0.

NEEDS TO BE CLEANED UP

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,181 @@
...
+- ImageBackground now respects `imageStyle` width and height ([dbd5c3d8e5](https://github.com/facebook/react-native/commit/dbd5c3d8e5e35685be89156194a96cead553a330) by [@Naturalclar](https://github.com/Naturalclar))
+- React Native sync for revisions 95d762e...e8feb11 ([88021894f2](https://github.com/facebook/react-native/commit/88021894f2645266d7e1b0a2e61a737b1590d039) by [@kacieb](https://github.com/kacieb))
+- React Native sync for revisions bd5bf55...95d762e ([7f7f8188f2](https://github.com/facebook/react-native/commit/7f7f8188f2ed3cb40b85e7f1ad65fdf31b678ffc) by [@rickhanlonii](https://github.com/rickhanlonii))
+- Remove defaultProps from the DrawerLayoutAndroid Component.�lunaleaps this is the fix for issue https://github.com/facebook/react-native/issues/31606 ([b998e32501](https://github.com/facebook/react-native/commit/b998e32501ab13525a1a7c711d5ad090e3d23d9f) by [@mdaj06](https://github.com/mdaj06))
+- Bump Hermes to 0.9.0 ([8e66f0b35b](https://github.com/facebook/react-native/commit/8e66f0b35b5ec1054994f7048ef3e66002544197) by [@Huxpro](https://github.com/Huxpro))
```
Comment (oblador) [inline CHANGELOG.md:51]: This was the highlight of 0.66.0 and is duplicated here, so should probably remove.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,181 @@
...
+- Changed the showcase URL ([1927ceb552](https://github.com/facebook/react-native/commit/1927ceb5520e36c5ca92e349ee1b860093290c15) by [@HeroBanana](https://github.com/HeroBanana))
+- ImageBackground now respects `imageStyle` width and height ([dbd5c3d8e5](https://github.com/facebook/react-native/commit/dbd5c3d8e5e35685be89156194a96cead553a330) by [@Naturalclar](https://github.com/Naturalclar))
+- React Native sync for revisions 95d762e...e8feb11 ([88021894f2](https://github.com/facebook/react-native/commit/88021894f2645266d7e1b0a2e61a737b1590d039) by [@kacieb](https://github.com/kacieb))
+- React Native sync for revisions bd5bf55...95d762e ([7f7f8188f2](https://github.com/facebook/react-native/commit/7f7f8188f2ed3cb40b85e7f1ad65fdf31b678ffc) by [@rickhanlonii](https://github.com/rickhanlonii))
+- Remove defaultProps from the DrawerLayoutAndroid Component.�lunaleaps this is the fix for issue https://github.com/facebook/react-native/issues/31606 ([b998e32501](https://github.com/facebook/react-native/commit/b998e32501ab13525a1a7c711d5ad090e3d23d9f) by [@mdaj06](https://github.com/mdaj06))
```
Comment (oblador) [inline CHANGELOG.md:50]:
```suggestion
- Remove defaultProps from the DrawerLayoutAndroid Component ([b998e32501](https://github.com/facebook/react-native/commit/b998e32501ab13525a1a7c711d5ad090e3d23d9f) by [@mdaj06](https://github.com/mdaj06))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,181 @@
...
+- Summary: Changelog: [Android][added] - Adds option use fallback bundle instead of cached bundle when the packager isn't running in ReactInstanceManager ([dd10134bdc](https://github.com/facebook/react-native/commit/dd10134bdcf9f046bde3136a694e3482e5876c34) by [@amy588](https://github.com/amy588))
+- Show Redbox for C++ errors. ([d6c879edba](https://github.com/facebook/react-native/commit/d6c879edbad068d0f461381875b7fae6db99d18d) by [@sota000](https://github.com/sota000))
+
+#### iOS specific
+
+- Add asdf-vm support in find-node.sh ([3e7c310b1d](https://github.com/facebook/react-native/commit/3e7c310b1dcf5643920535eea70afa451888792a) by [@pastleo](https://github.com/pastleo))
+- - [iOS][fixed] - ScrollView: Respect `contentInset` when animating new items with `autoscrollToTopThreshold`, make `automaticallyAdjustKeyboardInsets` work with `autoscrollToTopThreshold` (includes vertical, vertical-inverted, horizontal and horizontal-inverted ScrollViews) ([6e903b07fa](https://github.com/facebook/react-native/commit/6e903b07fa8e8d9b78cae0e031bb8022f7a63195) by [@mrousavy](https://github.com/mrousavy))
```
Comment (oblador) [inline CHANGELOG.md:37]:
```suggestion
- ScrollView: Respect `contentInset` when animating new items with `autoscrollToTopThreshold`, make `automaticallyAdjustKeyboardInsets` work with `autoscrollToTopThreshold` (includes vertical, vertical-inverted, horizontal and horizontal-inverted ScrollViews) ([6e903b07fa](https://github.com/facebook/react-native/commit/6e903b07fa8e8d9b78cae0e031bb8022f7a63195) by [@mrousavy](https://github.com/mrousavy))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,181 @@
...
+- Remove defaultProps from the DrawerLayoutAndroid Component.�lunaleaps this is the fix for issue https://github.com/facebook/react-native/issues/31606 ([b998e32501](https://github.com/facebook/react-native/commit/b998e32501ab13525a1a7c711d5ad090e3d23d9f) by [@mdaj06](https://github.com/mdaj06))
+- Bump Hermes to 0.9.0 ([8e66f0b35b](https://github.com/facebook/react-native/commit/8e66f0b35b5ec1054994f7048ef3e66002544197) by [@Huxpro](https://github.com/Huxpro))
+- Bump Hermes to 0.9.0 ([66bd1e9c2c](https://github.com/facebook/react-native/commit/66bd1e9c2c5c12d8652847b068ea149e790259cf) by [@Huxpro](https://github.com/Huxpro))
+- React-native/normalize-color now supports Node.js ([65e58f26e1](https://github.com/facebook/react-native/commit/65e58f26e1fbd06b1ae32e2ab3a2616c8eef08e0) by [@yungsters](https://github.com/yungsters))
+- Updated to Contributor Covenant v2.1 ([19f8d2f7da](https://github.com/facebook/react-native/commit/19f8d2f7da13f4524f31acf7aa10cc0aa91b5da4))
+- Added snapshot test for the new component ([2fb102b601](https://github.com/facebook/react-native/commit/2fb102b601a929be74eb9042745a1bb63ed7bc6a))
```
Comment (oblador) [inline CHANGELOG.md:55]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,181 @@
...
+
+#### Android specific
+
+- Bump Gradle version to 7.2 ([9ae3367431](https://github.com/facebook/react-native/commit/9ae3367431428748f5486c782199beb4f9c6b477) by [@svbutko](https://github.com/svbutko))
+- Add some logs ([24ac66984c](https://github.com/facebook/react-native/commit/24ac66984c2d0c55ae09fb2df0f1e88a42339f21))
```
Comment (oblador) [inline CHANGELOG.md:60]:
```suggestion
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,181 @@
...
+
+- Breaking: Removed initialization of React Native polyfills and global variables from React renderers. ([b14b34b232](https://github.com/facebook/react-native/commit/b14b34b232b4873137ea3f3c7a90f86b9013885a) by [@rubennorte](https://github.com/rubennorte))
+
+#### Android specific
+
+#### iOS specific
+
+### Added
+
+- Make react-native depend on react-native-gradle-plugin ([80e24834ba](https://github.com/facebook/react-native/commit/80e24834ba8c581157fb6c9f5d8b49301677c487) by [@cortinico](https://github.com/cortinico))
```
Comment (cortinico) [inline CHANGELOG.md:15]: This change was reverted in a subsequent commit, so maybe should be placed lower in the list or removed.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,181 @@
...
+- Added examples to Switch component ([c9c14ef687](https://github.com/facebook/react-native/commit/c9c14ef6872eb90b7305e0bc19cded76ce851186) by [@suminkimm](https://github.com/suminkimm))
+- Added `cancelButtonTintColor` prop for `ActionSheetIOS` to change only the text color of the cancel button ([01856633a1](https://github.com/facebook/react-native/commit/01856633a1d42ed3b26e7cc93a007d7948e1f76e) by [@nomi9995](https://github.com/nomi9995))
+- Added [LSApplicationQueriesSchemes](https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/LaunchServicesKeys.html#//apple_ref/doc/uid/TP40009250-SW14) in info.plist with entries tel, telprompt, http, fb, geo ([b26f277262](https://github.com/facebook/react-native/commit/b26f2772624c863c91fa1ff627b481c92d7562fb) by [@utkarsh-dixit](https://github.com/utkarsh-dixit))
+- Adding Accessibility Display Options Enabled to RNTester ([2abacace54](https://github.com/facebook/react-native/commit/2abacace54dc72bac1a5d7f2fb795edd89374b1c) by [@graytmatterMS](https://github.com/graytmatterMS))
```
[the remark this answers is not in the document]
Comment (kelset) [reply CHANGELOG.md:16]: added here: https://github.com/react-native-community/releases/issues/257

**Comment by kelset** [comment]
Closing this in favour of https://github.com/facebook/react-native/pull/32529, the feedback on the generator has been recorded here to make sure we won't lose it: https://github.com/react-native-community/releases/issues/257

**Pull request closed without merging**